tests/integration/bulk_import/test_mal_annotation_import.py

```diff
@@ -0,0 +1,161 @@
+#
+# This was commented out, as it is almost ready to go, however we do not yet have an updated
+# implementation of project.upload_annotations that uses AnnotationImports rather than
+# BulkImportRequest. Once we have that, we should be calling this new implementation
+# rather than the old one.
```

Comment:
I thought about this some more. We should just put this in another branch instead of committing this commented out code.

Reply:
Fine with me; sure.

Reply:
Heres the branch `grant/mal-annotation-import-test-PR`